vscode/src/completions/providers/fireworks.ts

```diff
@@ -83,8 +95,10 @@ const MODEL_MAP = {
     'llama-code-13b': 'fireworks/accounts/fireworks/models/llama-v2-13b-code',
 
     // Fine-tuned model mapping
-    'fireworks-completions-fine-tuned':
-        'fireworks/accounts/sourcegraph/models/codecompletion-mixtral-rust-152k-005e',
+    'fim-fine-tuned-model-variant-1': FIREWORKS_FIM_FINE_TUNED_MODEL_1,
```

Comment:
Since we have constants already, we can: 
```suggestion
    [FIREWORKS_FIM_FINE_TUNED_MODEL_1]: FIREWORKS_FIM_FINE_TUNED_MODEL_1,
```

Reply:
changed